setup/steps/configure_log_min_duration_statement.go

```diff
@@ -0,0 +1,57 @@
+package steps
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	survey "github.com/AlecAivazis/survey/v2"
+	"github.com/pganalyze/collector/setup/state"
+	s "github.com/pganalyze/collector/setup/state"
+	"github.com/pganalyze/collector/setup/util"
+)
+
+var ConfigureLogMinDurationStatement = &s.Step{
+	Kind:        state.LogInsightsStep,
+	Description: "Check log_min_duration_statement",
+	Check: func(state *s.SetupState) (bool, error) {
+		row, err := state.QueryRunner.QueryRow(`SELECT setting FROM pg_settings WHERE name = 'log_min_duration_statement'`)
+		if err != nil {
+			return false, err
+		}
+
+		lmdsVal := row.GetInt(0)
+		needsUpdate := (lmdsVal < 10 && lmdsVal != -1) ||
```

Comment:
Could we get the constant (`10`) from the main collector instead? (so we don't forget to update it)

(The same probably applies for `auto_explain.log_min_duration`)